extensions/websockets-next/runtime/src/main/java/io/quarkus/websockets/next/runtime/telemetry/WebsocketTelemetryProviderBuilder.java

```diff
@@ -0,0 +1,98 @@
+package io.quarkus.websockets.next.runtime.telemetry;
+
+import java.util.List;
+import java.util.Objects;
+import java.util.function.Function;
+
+import io.quarkus.websockets.next.runtime.WebSocketEndpoint;
+import io.quarkus.websockets.next.runtime.telemetry.ConnectionInterceptor.CompositeConnectionInterceptor;
+
+/**
+ * Quarkus uses this class internally to build {@link WebsocketTelemetryProvider}.
+ */
+public final class WebsocketTelemetryProviderBuilder {
```

Comment:
```suggestion
public final class WebSocketTelemetryProviderBuilder {
```
Just for consistency...